db/migrations/20221226104121_create_order_items_table.sql

```diff
@@ -0,0 +1,18 @@
+-- migrate:up
+CREATE TABLE order_items (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    user_id INT NOT NULL,
+    book_id INT NOT NULL,
+    bookshelf_id INT NOT NULL,
+    order_status_id INT NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
+    deleted_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
```

Comment:
update 될 때 deleted_at 컬럼을 수정하는 것이 아닌, 실제 삭제 method로 요청이 들어왔을 때 해당 시간을 기입해주는 방법은 어떨까요?

```suggestion
    deleted_at TIMESTAMP NULL,
```